v63003/inref/gtm8804.m

```diff
@@ -0,0 +1,38 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+; Tests that ZSHOW "T" produces summary lines for "G" and "L"
+;
+
+gtm8804
+	set ^x=1
```

Comment:
Add a comment that this is done to ensure ZSHOW "G" has at least one non-zero statistic. Likewise, add a comment that line 18 is done to ensure ZSHOW "L" has at least one non-zero statistic.